notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb

```diff
@@ -615,7 +558,7 @@
       "source": [
         "#### Task.py contents\n",
         "\n",
-        "In the next cell, you write the contents of the training script task.py. We won't go into detail, it's just there for you to browse. In summary:\n",
+        "In the next cell, you write the contents of the training script task.py. You don't have to go into detail. It's just there for you to browse. In summary:\n",
```

Comment:
Change to:

- In the next cell, provide the contents of the training script task.py file. You don't have to go into detail. ..."

Reply:
Did required changes